src/client/components/pages/auth/styles.scss

```diff
@@ -47,3 +47,4 @@
     text-align: center;
   }
 }
+}
```

Comment:
this change seems unnecessary. 

PROTIP: whenever u can a developer should create PR with the less module changes as possible. (without this change the PR will just have 9 modules changed) (less is better)